src/patternfly/components/CustomSearchInput/custom-search-input.scss

```diff
@@ -0,0 +1,120 @@
+  --pf-c-custom-search-input__icon--TranslateY: -50%;
+
+  // Utilities
+  --pf-c-custom-search-input__utilities--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-custom-search-input__utilities--child--MarginLeft: var(--pf-global--spacer--xs);
+  --pf-c-custom-search-input__utilities--c-button--PaddingRight: var(--pf-global--spacer--xs);
+  --pf-c-custom-search-input__utilities--c-button--PaddingLeft: var(--pf-global--spacer--xs);
+
+  position: relative;
+  display: flex;
+  padding: var(--pf-c-custom-search-input--PaddingTop) var(--pf-c-custom-search-input--PaddingRight) var(--pf-c-custom-search-input--PaddingBottom) var(--pf-c-custom-search-input--PaddingLeft);
+
+  &:hover {
+    --pf-c-custom-search-input__text--after--BorderBottomColor: var(--pf-c-custom-search-input--hover__text--after--BorderBottomColor);
+  }
+}
+
+.pf-c-custom-search-input__text {
+  flex: 1;
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    pointer-events: none;
+    content: "";
+  }
+
+  &::before {
+    border: var(--pf-c-custom-search-input__text--before--BorderWidth) solid var(--pf-c-custom-search-input__text--before--BorderColor);
+  }
+
+  &::after {
+    border-bottom: var(--pf-c-custom-search-input__text--after--BorderBottomWidth) solid var(--pf-c-custom-search-input__text--after--BorderBottomColor);
+  }
+
+  &:hover {
+    --pf-c-custom-search-input__icon--Color: var(--pf-c-custom-search-input__text--hover__icon--Color);
+  }
+
+  &:focus-within {
+    --pf-c-custom-search-input__text--after--BorderBottomWidth: var(--pf-c-custom-search-input__text--focus-within--after--BorderBottomWidth);
+    --pf-c-custom-search-input__text--after--BorderBottomColor: var(--pf-c-custom-search-input__text--focus-within--after--BorderBottomColor);
+  }
+}
+
+.pf-c-custom-search-input__icon {
+  position: absolute;
+  top: 50%;
+  left: var(--pf-c-custom-search-input__icon--Left);
+  color: var(--pf-c-custom-search-input__icon--Color);
+  transform: translateY(var(--pf-c-custom-search-input__icon--TranslateY));
+}
+
+.pf-c-custom-search-input__text-input {
+  @include pf-text-overflow;
+
+  position: relative;
+  width: 100%;
+  min-width: var(--pf-c-custom-search-input__text-input--MinWidth);
+  padding: var(--pf-c-custom-search-input__text-input--PaddingTop) var(--pf-c-custom-search-input__text-input--PaddingRight) var(--pf-c-custom-search-input__text-input--PaddingBottom) var(--pf-c-custom-search-input__text-input--PaddingLeft);
+  border: 0;
+
+  &:hover,
+  &:focus {
+    + .pf-c-custom-search-input__icon {
+      --pf-c-custom-search-input__icon--Color: var(--pf-c-custom-search-input__text-input--focus--icon--Color);
+    }
+  }
+}
+
+.pf-c-custom-search-input__utilities {
+  display: flex;
+  margin-right: var(--pf-c-custom-search-input__utilities--MarginRight);
+
+  > * + * {
+    margin-left: var(--pf-c-custom-search-input__utilities--child--MarginLeft);
+  }
```

Comment:
Then you wouldn't need to use the lobotomized owl.. 😆

Reply:
I love the owl! Yeah I used flex in case the column widths change, we have a var we can manage unique spacing. We could still manage the spacing using grid, but it would be a combination of `grid-gap` and a margin. wdyt? I don't feel strongly about it either way.